api/middleware.js

```diff
@@ -162,7 +162,21 @@ const parseArray = function(field) {
 };
 
 const requestWrapper = fn => (request, response, next) => {
-  log.info(format('%s Request: %s', request.method, request.url));
+  if (request.user) {
+    log.info(format('%s %s (user: %s)'),
+      request.method,
+      request.url,
+      request.user.get("username"),
+    );
+  } else if (request.device) {
+    log.info(format('%s %s (device: %s)'),
+      request.method,
+      request.url,
+      request.device.get("devicename"),
+    );
+  } else {
+    log.info(format('%s %s', request.method, request.url));
+  }
```

Comment:
Given the `<method> <URL>` part is always the same, how about creating that first and then using it log calls in the user, device, not-logged-in branches?

This makes the code more concise and avoids unintentional differences creeping in as the code evolves.